src/main/java/com/synerzip/projectmanagementapp/controller/EmployeeController.java

```diff
@@ -13,50 +14,52 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.codehaus.jackson.JsonGenerationException;
+import org.codehaus.jackson.map.JsonMappingException;
+import org.codehaus.jackson.map.ObjectMapper;
 
 import com.synerzip.projectmanagementapp.model.Employee;
+import com.synerzip.projectmanagementapp.model.PageResult;
 import com.synerzip.projectmanagementapp.serviceimplementation.EmployeeServicesImplementation;
 
 @Path("/employee")
 public class EmployeeController {
 
 	EmployeeServicesImplementation service = new EmployeeServicesImplementation();
 
-	@GET
-	@Produces(MediaType.APPLICATION_JSON)
-	@Path("{empId}")
+	@GET@Produces(MediaType.APPLICATION_JSON)@Path("/{empId}")
 	public Employee getEmployee(@PathParam("empId") long empId) {
 		return service.getEmployee(empId);
 	}
 
-	@GET
-	@Produces(MediaType.APPLICATION_JSON)
-	public List<Employee> getEmployees(
```

Comment:
as these all methods in EmployeeController so no need to append employee with method name. Its will looks if you use get method name instead of getEmployee. Same comment valid for all controller.